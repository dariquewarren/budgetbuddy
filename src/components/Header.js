import React from 'react'
import {BrowserRouter, Route, Switch, Link, NavLink} from 'react-router-dom'

const Header = ()=> (
    <header>
    <h1>Budget Buddy</h1>
  <NavLink to="/" activeClassName="is-active" exact={true}>Home Sweet Home</NavLink>
  <NavLink to="/create" activeClassName="is-active">Create Expense</NavLink>
  <NavLink to="/edit" activeClassName="is-active">Edit Expenses</NavLink>
  <NavLink to="/help" activeClassName="is-active">Help Page</NavLink>
    </header>
    )


export default Header 