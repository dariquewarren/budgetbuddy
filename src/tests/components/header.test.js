// react test renderer
import React from "react";
import { shallow } from "enzyme";
import Header from "../../components/Header";

test("should render headercmpnt", () => {
  const wrapper = shallow(<Header />);
// console.log(wrapper.props())

    expect(wrapper.props()).toMatchSnapshot();
  // expect(wrapper.find('h1').text()).toBe('Budget Buddy')
  // const renderer = new ReactShallowRenderer()
  // renderer.render(<Header/>)
  // expect(renderer.getRenderOutput()).toMatchSnapshot()
});
