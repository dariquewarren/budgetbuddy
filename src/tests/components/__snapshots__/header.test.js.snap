// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render headercmpnt 1`] = `
Object {
  "children": Array [
    <h1>
      Budget Buddy
    </h1>,
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      exact={true}
      to="/"
    >
      Dashboard
    </NavLink>,
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      to="/create"
    >
      Create Expense
    </NavLink>,
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      to="/help"
    >
      Help Page
    </NavLink>,
  ],
}
`;
